apps/docs/content/guides/platform/pause-restore.mdx

```diff
@@ -0,0 +1,43 @@
+---
+id: 'pause-restore'
+title: 'Pausing and Restoring'
+description: 'Pausing and restoring projects.'
+---
+
+Organizations on a Free Plan can [pause](/dashboard/project/_/settings/general) and restore projects. This can be done to conserve resources for unused projects, or to upgrade a project to the latest version of each service.
+
+<Admonition type="note">
+
+Free projects are automatically paused after <SharedData data="config">pausing.free_tier</SharedData> of inactivity.
+
+</Admonition>
+
+## Restoring a project
+
+A backup is created when a project is paused. You can restore the backup from the dashboard within 90 days. After 90 days, you can download the backup file to manually restore or extract its data.
+
+The 90-day window allows Supabase to introduce platform changes that may not be backwards compatible with older backups. Unlike active projects, static backups can't be updated to accommodate such changes.
+
+### 90-day restore window
+
+During the 90-day restore window, you can restore a paused project from [your project dashboard](/dashboard/projects).
+
+<Image alt="Project Paused: 90 Days Remaining" src="/docs/img/guides/platform/paused-90-day.png" />
+
+### Post-restore window
+
+After the 90-day restore window, you can download your project's backup file from the project dashboard. This backup is compatible with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html). You can extract your data from it, or attempt to manually restore the project directly.
```

Comment:
Mention that pgsodium-managed data isn't handled?

Reply:
Wouldn't they receive the encrypted data and then hit the management api to retrieve their encryption key?

Even if its a multi-file backup we should still provide it to them and provide minimal instructions on how to restore using it

Reply:
I'm going to merge without this change to unblock everything else but will create a separate PR to update the docs with your feedback

Reply:
Yeah I think the part that's missing right now is more detailed instructions (e.g. a guide) on how they can restore the backup that they download.
Pretty sure right now the link is for a single-file backup, which would double-encrypt on restore (unless we have instructions to edit it to disable triggers etc, quite an involved process)